build.sbt

```diff
@@ -21,17 +21,18 @@ organization := "com.spotify"
 name := "big-data-rosetta-code"
 version := "0.1.0-SNAPSHOT"
 
-val scioVersion = "0.6.1"
+val scioVersion = "0.7.0"
 val scaldingVersion = "0.17.4"
 val sparkVersion = "2.4.0"
-val algebirdVersion = "0.13.0"
+val algebirdVersion = "0.13.5"
 val scalacheckVersion = "1.14.0"
 val scalameterVersion = "0.10"
 
-scalaVersion := "2.11.12"
+scalaVersion := "2.12.8"
 scalacOptions ++= Seq("-target:jvm-1.8", "-deprecation", "-feature", "-unchecked")
 javacOptions ++= Seq("-source", "1.8", "-target", "1.8")
 
+resolvers += "Cascading libraries" at "http://conjars.org/repo"
```

Comment:
@ClaireMcGinty did you tried `Resolver.sonatypeRepo("public")`?

Reply:
I tried, but still wasn't able to resolve Cascading libraries. I don't think they're available in Sonatype public repos: https://oss.sonatype.org/content/repositories/public/

Reply:
yeah I was just checking 😄